packages/angular_devkit/build_angular/test/browser/differential_loading_spec_large.ts

```diff
@@ -63,6 +63,36 @@ describe('Browser Builder with differential loading', () => {
       .toEqual(jasmine.arrayWithExactContents(expectedOutputs));
   });
 
+  it('deactivates differential loading for watch mode', async () => {
+    const { files } = await browserBuild(architect, host, target, { watch: true });
+
+    const expectedOutputs = [
```

Comment:
Small nit: we should emit es2015 builds and not es5 to be consistent with server, e2e, test and all the other builders